README.md

```diff
@@ -804,7 +804,7 @@ function isAdmin(usr: Admin | User): usr is Admin {
 }
 ```
 
-If you need `if/elseif` chains or the `switch` statement instead, it should "just work", but look up [Discriminated Unions](https://www.typescriptlang.org/docs/handbook/advanced-types.html) if you need help. (See also: [Basarat's writeup](https://basarat.gitbooks.io/typescript/docs/types/discriminated-unions.html)). This is handy in typing reducers for `useReducer` or Redux.
+If you need `if...else` chains or the `switch` statement instead, it should "just work", but look up [Discriminated Unions](https://www.typescriptlang.org/docs/handbook/advanced-types.html) if you need help. (See also: [Basarat's writeup](https://basarat.gitbooks.io/typescript/docs/types/discriminated-unions.html)). This is handy in typing reducers for `useReducer` or Redux.
```

Comment:
Suggesting `if...else` instead of `if/elseif` to follow the semantics of [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else#Description)